filters/include/pcl/filters/impl/voxel_grid_covariance.hpp

```diff
@@ -352,6 +352,7 @@ pcl::VoxelGridCovariance<PointT>::applyFilter (PointCloud &output)
         leaf.cov_ = leaf.evecs_ * eigen_val * leaf.evecs_.inverse ();
       }
       leaf.evals_ = eigen_val.diagonal ();
+      leaf.covdet_ = leaf.cov_.determinant ();
```

Comment:
Should we move this post the next if condition (and add a continue in the condition)?